_books/ion-1-1/src/modules/encoding_modules.md

```diff
@@ -0,0 +1,228 @@
+# Encoding modules
+
+The encoding of each segment of a stream is shaped by the currently configured _encoding modules_,
+an ordered sequence of modules that determine which symbols and macros are available for use in the stream.
+A writer can modify this sequence by emitting an [encoding directive](directives.md#encoding-directives).
+
+By logically concatenating the encoding modules' symbol and macro tables respectively,
+they can be viewed as unified local symbol and macro tables.
+
+For example, consider this encoding directive:
+```ion
+$ion::
+(encoding
```

Comment:
I think we can simplify things both in terms of implementations and also in terms of mental model if we say that the `encoding` directive cannot create new bindings—it can only specify an ordering of existing bindings.